src/stylesheets/base/_print.scss

```diff
@@ -0,0 +1,101 @@
+@media print {
+  // Ensure elements have transparent backgrounds and text is black
+  // plus remove any shadow effects
+  *,
+  *:before,
+  *:after,
+  *:first-letter,
+  p:first-line,
+  div:first-line,
+  blockquote:first-line,
+  li:first-line {
+    background: transparent !important;
+    color: #000 !important;
+    box-shadow: none !important;
+    text-shadow: none !important;
+  }
+  // Set body and article elements to 100% width and remove margins
+  body,
+  article {
+    width: 100% !important;
+    margin: 0 !important;
+    padding: 0 !important;
+  }
+  // Hide elements that are not necessary for print
+  nav,
+  aside,
+  footer {
+    display: none !important;
```

Comment:
This completely hides breadcrumbs and eventually pagination on print.

![image](https://user-images.githubusercontent.com/3385219/109563241-20001180-7aa5-11eb-9776-f85d51ba5875.png)